scripts/processAndDownloadContracts.ts

```diff
@@ -17,21 +17,25 @@ async function processContracts() {
 			...await getAaveV2Misc()
 		]
 	const output = `
-
-const contractData: [string, string, string, string | null][] = ${JSON.stringify(contractData.map((x) => [addressString(x.address), x.data.name, x.data.protocol, x.data.logoURI]), null, 1)}
+import * as contractData from './contractMetadata.json';
 export type ContractDefinition = {
-    name: string,
-    logoURI: string | undefined,
-    protocol: string | undefined
+	name: string,
+	logoURI: string | undefined,
+	protocol: string | undefined
 }
 export const contractMetadata = new Map<string, ContractDefinition>(
```

Comment:
Why is this a `Map` instead of just an object?  I think we already talked about this but I *thought* we decided to go with objects?